website/src/controllers/api-v2/members.js

```diff
@@ -38,7 +38,7 @@ api.sendMessage = function(user, member, data){
     msg = data.message
   } else {
     msg = "`Hello " + member.profile.name + ", " + user.profile.name + " has sent you ";
-    msg += (data.type=='gems') ? data.gems.amount + " gems!`" : shared.content.subscriptionBlocks[data.subscription.key].months + " months of subscription!`";
+    msg += (data.type=='gems') ? data.gems.amount + " gem" + ((data.gems.amount>1)?"s":"") + "!`" : shared.content.subscriptionBlocks[data.subscription.key].months + " month" + ((shared.content.subscriptionBlocks[data.subscription.key].months>1)?"s":"") + " of subscription!`";
```

Comment:
When gems are gifted, the message appears in the sender's mailbox as well as the receiver's, so the sender needs to be able to read it, so sometimes two languages would be needed.

I'm wondering if both players should see the message in both languages, so that they still get identical copies.

I think the translation should be done in a separate issue since it's not trivial.